python-engine/yggdrasil_engine/engine.py

```diff
@@ -182,6 +185,11 @@ def take_state(self, state_json: str) -> Optional[List[Warning]]:
                 return warnings
             return None
 
+    def get_state(self) -> str:
+        response_ptr = self.lib.get_state(self.state)
+        with self.materialize_pointer(response_ptr, dict) as result:
+            return json.dumps(result.value)
```

Comment:
Same as Ruby I think we'll want to unwrap the wrapping, so returning `result['value']` is probably what we want